recipes/pyvtk/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.5.18" %}
+{% set name = "PyVTK" %}
+{% set sha256 = "608b509b0da4525ad43078b3a60185c359c16f07a2f6b44fbd6f28fb4b9c4e53" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
```

Comment:
Done and pushed - let's see (I don't know why it wants to use numpy 1.9 for pyvtk.)